ocs_ci/ocs/fio_artefacts.py

```diff
@@ -123,7 +123,7 @@ def get_job_dict():
     if arch.startswith("x86"):
         image = "quay.io/fbalak/fio-fedora:latest"
     else:
-        image = "quay.io/multiarch-origin-e2e/fio-fedora:latest"
+        image = "quay.io/multi-arch/fio-fedora:latest"
```

Comment:
try to use quay.io/ocsci/perf:latest
this is multi architecture image with FIO installed in it (based on alpine), and you can just consume it without checking the arch.

Reply:
is that a drop in replacement?  is there any reason not to use that for *all* architectures, including x86?

Reply:
Module `fio_artefacts` assumes the code is using recent fedora fio package, with http I/O engine enabled (which is enabled since [fio-3.19-3.fc32](https://bugzilla.redhat.com/show_bug.cgi?id=1836323)).

When this `if` statement was inserted about a year ago, Filip was able to build x86 container image only and the change haven't reached Fedora yet. As long as the container https://quay.io/repository/multi-arch/fio-fedora contains latest fedora rpm package, it should be fine to use single fio fedora multiarch image and drop this if statement.

We need to have full control over fio build for debugging purposes and for us to be able to patch fio if necessary (these will end up upstreamed eventually). I would like multi-arch/fio-fedora image has metadata specified properly though.

Reply:
> try to use quay.io/ocsci/perf:latest

Not here.

Reply:
I tried to use the multiarch image and I'm getting:

```
Failed to pull image "quay.io/multi-arch/fio-fedora:latest": rpc error: code = Unknown desc = Error choosing image instance: no image found in manifest list for architecture amd64, variant "", OS linux
```

So the `if` definitely needs to stay as it is for now. No changes in the pull request needed.